rules/default.js

```diff
@@ -50,10 +49,7 @@ module.exports = function (options) {
 
     heading : function (text, level, raw) {
         var result = {
-            elem : 'header',
-            mods : { // BUG: MUST BE elemMods
-                level : level
-            },
+            elem : 'h' + level,
```

Comment:
maybe add:
```
mix: {
    elem: 'header',
}
```
or
```
mix: {
    elem: 'h',
}
```
for writing common css for all headers, for example — common font-family